src/main/java/io/stargate/sgv2/jsonapi/service/operation/keyspaces/FindKeyspacesOperation.java

```diff
@@ -0,0 +1,54 @@
+package io.stargate.sgv2.jsonapi.service.operation.keyspaces;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import io.smallrye.mutiny.Uni;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandResult;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandStatus;
+import io.stargate.sgv2.jsonapi.api.request.DataApiRequestInfo;
+import io.stargate.sgv2.jsonapi.service.cqldriver.CQLSessionCache;
+import io.stargate.sgv2.jsonapi.service.cqldriver.executor.QueryExecutor;
+import io.stargate.sgv2.jsonapi.service.operation.Operation;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Supplier;
+
+/**
+ * Operation that list all available keyspaces into the {@link CommandStatus#EXISTING_KEYSPACES}
+ * command status.
+ *
+ * @param cqlSessionCache CQLSession cache for keyspace fetching
+ */
+public record FindKeyspacesOperation(CQLSessionCache cqlSessionCache) implements Operation {
```

Comment:
-1 on having this class and FindNamespacesOperation with the same code except for 1 line. 

Recommend:
* make this a class, it was a record from peeps get too excited about records. Remove the CQLSessionCache it is not needed
* add a ctor param  useKeyspaceName and use that when creating the result. 
* Get the session case from the QueryExecutor , that is what it is there for and it already has a method on it. 
* Keep the two command resolvers, they only work with a  single command class, but they can both create the same operation,.